docs/UserGuide.md

```diff
@@ -313,3 +320,4 @@ and paste it in your other computer in the same directory where YMFC.jar is loca
 | `delete n/NAME`                                                 | Delete the recipe of that name                                                                            |
 | `new n/INGREDIENT`                                              | Add a new ingredient to your list                                                                         |
 | `listI`                                                         | List out all your current ingredients                                                                     |
+| `findI INGREDIENT`                                              | Searches ingredient list with input ingredient name                                                       |                                             
```

Comment:
I think we should change from "Searches" to "Search" to maintain the consistency among other commands' descriptions.

Reply:
I concur